lib/bundler/stats/cli.rb

```diff
@@ -70,6 +84,20 @@ def draw_show(stats, target)
         say ""
       end
 
+      def draw_versions(stats, target)
+        say "bundle-stats for #{target}"
+        say ""
+        say "depended upon by (#{stats[:top_level_dependencies].count}):\n"
+        say "+------------------------------|-------------------+"
```

Comment:
Could probably skip printing the table if `count == 0` (no dependencies) 😄

Reply:
Good suggestion @gmcnaughton, I may take a minute to get to this, but I'll merge a PR if you get a chance to grab it first.